### DISCUSSION TITLE
Hacktober is Here 👻 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#69241] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret, striped shirt, and red pants holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

While the end of the month may bring trick or treating, the real treat is all month long with Hacktoberfest! 

What is Hacktoberfest? Digitalocean’s annual event in its TENTH YEAR that celebrates open source contributions, giving back to projects that don’t have the resources to expand, and expanding your skills ✏️. There are ways to get involved even with low/non-code contributions as well. 

If you’re new to the open source/development world this is a fun way to get started and explore what is out there! [Sign up and get started](https://hacktoberfest.com/auth/). 

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @linuxlawson & @satilpereira: For both of their help in answering how to [show language on your repository](https://github.com/orgs/community/discussions/68947). Two heads are better than one 👯. 
* @vishalkrishna8: For the [refresher](https://github.com/orgs/community/discussions/68745) on how to get back into the swing of GitHub if you’ve been away for a moment. Your time spent on GitHub may come and go, but our community lending a helping hand is constant. 
* @thaiknguyen: Came armed with feedback and a workaround in [their post](https://github.com/orgs/community/discussions/68264) - thanks for sharing about having a default permissions for all repos in an org. 

### 🧡 Hacktober Resources: Get in the virtual spirit and connect with fellow devs 🍁
* **Participating Hacktoberfest Projects on GitHub**: Find GitHub-hosted projects to celebrate through [this collection](https://github.com/topics/hacktoberfest). 
* **How to Contribute to Open Source**: Ready to contribute to open source? Check out [this guide](https://opensource.guide/how-to-contribute/) on how to get started. 
* **Hacktoberfest Guide Repository**: [A curated list](https://github.com/OtacilioN/awesome-hacktoberfest) of awesome Hacktoberfest 2023 repositories, guides and resources once again put together by @OtacilioN ❣️. 

### 🎃 Hacktober Events: Get in the virtual spirit and connect with fellow devs 🍭
* Hacktoberfest Kickoff: Learn the history of Hacktoberfest, how to participate, what's new this year, and how to get started on October 5th at 11am EST. [Register here](https://events.mlh.io/events/10248-official-digitalocean-hacktoberfest-10th-anniversary-kickoff?_gl=1*xu0rqo*_ga*MTAwMjM2NzI0MS4xNjg5MDEwNDQ3*_ga_E5KT6TC4TK*MTY5NTIzMDIwNy41Mi4xLjE2OTUyMzAyMTQuMC4wLjA.). 
* **Global Hack Week**: Major League Hacking is spending October 16th-23rd exploring Open Source. Join them to learn how to start making contributions to Open Source projects! [Register and find out more information](https://ghw.mlh.io/events/open-source)
* **HacktoberWeek 2.0**: Starting October 14, spend seven days learning Git/GitHub, joining interactive sessions, and getting practical real world experience. [Find out the details and sign up to join](https://events.mlh.io/events/10386-hacktoberweek-2-0). 

What projects are you working on for Hacktoberfest? Celebrate with us by sharing below.

**Comment by d-coder111** [comment]
Hey @queenofcorgis 👋
I want my repository to be considered under hactoberfest, what should i do for the same?

**Reply by queenofcorgis** [reply]
@d-coder111 Digitalocean is the host of Hacktoberfest and their website has the details for how to participate. [Review the maintainers section for a guide](https://hacktoberfest.com/participation/#maintainers).

**Reply by d-coder111** [reply]
Sure, thanks.

**Comment by HarshanaJayasekara** [comment]
Respones #1

**Comment by queenofcorgis** [comment]
More Hacktober events, Universe, and more goodies [in the latest check-in](https://github.com/orgs/community/discussions/70498)!
